cms/envs/common.py

```diff
@@ -41,7 +41,7 @@
     # indirectly accessed through the email opt-in API, which is
     # technically accessible through the CMS via legacy URLs.
     PROFILE_IMAGE_BACKEND, PROFILE_IMAGE_DEFAULT_FILENAME, PROFILE_IMAGE_DEFAULT_FILE_EXTENSION,
-    PROFILE_IMAGE_SECRET_KEY, PROFILE_IMAGE_MIN_BYTES, PROFILE_IMAGE_MAX_BYTES,
+    PROFILE_IMAGE_SECRET_KEY, PROFILE_IMAGE_MIN_BYTES, PROFILE_IMAGE_MAX_BYTES, ENABLE_CREDIT_ELIGIBILITY
```

Comment:
the comment above this group of imported names doesn't apply to `ENABLE_CREDIT_ELIGIBILITY`.  Please separate and add a relevant comment.